.travis.yml

```diff
@@ -0,0 +1,39 @@
+addons:
+  apt:
+    packages:
+      - expect-dev
+
+language: python
+
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+  - "3.6"
+
+env:
+  global:
+    - VERSION="0.0.1"
+    - RELEASE="0.1.0"
+    - PROJECT="oca-decorators"
+    - ODOO="10.0"
+    - secure: "N4FxLhiQYktu5TUym48Kf5GoCeqSn/iNZT4rK9ktN18gOazkj5P9tPz33p8vde06f+yDTw6ETOZaTHUv7oGwlpc58ToMFey+XxlSyCeGoNVGi7uojs5DqF48lfKZbbTjCiiyj/6Ub7FH3IzcizbN+9WzJ6JB7v8U6eX8LwAW7YKPhdPctFM7rFZiLQp/r8lf0xN5WpMhufg3xloBJkS+iuQopULK7Il2g6Ag8Qo54IUFJzWQbZYKN8RMy0Ek41rgx88sKrTp9TDo0RBircy7f/6WH/CERTGRYtHCQDF0RToYaH9qiRiZ3w6vaxepbIicJCMCMvOQaSJ8IibIwBstwpkM0P2dHVP9WQiwxGgYMeCxS2MNkacQLAfxGXcMiRlfliq7SQpBYp7t3uEpby1l1UqDyTb+5aNUF8Y5CFx1hbbr2r++U8vLRxxvQRAzfl48Dy1jWw5N1hug/1VbGU7XuWPHiZd2WRngs2r7kUJVig9b3jQoi87kIr0V8FHjRALfsfo/swPD5mwI00qQ2eOySTKC8SSCToudByub1DYAPvuLn0JDGH1z7pDIe4rucZSSztvkiCOtlndYcm2hbyNLLVUN5Fn/GCmNZp2yer/h5Myn2kw3bMjdDLkVcgK5Ku2/eb5IMUWrjL4MbdczsUeqfNRK9oy5HEAAh+4sfre5Dw0="
```

Comment:
Generate a token for the OCA PyPi account

Also Travis needs secure variable for a Github token to post gh-pages